### ISSUE TITLE
N-API support for Node.js 8

### ISSUE THREAD

**Issue by vshymanskyy** [node-ffi/node-ffi#407]
Are there any plans to support N-API? I think it can **drastically** simplify binary packages deployment ;)

https://medium.com/the-node-js-collection/n-api-next-generation-node-js-apis-for-native-modules-169af5235b06

**Comment by TooTallNate** [comment]
Yes, definitely plans. Not sure when I'll find the time for the refactor though. Patches always welcome!

**Comment by happilymarrieddad** [comment]
+1
